arm-templates/azuredeploy-ctm.parameters.json

```diff
@@ -32,7 +32,7 @@
         "disqualifierImageTag": {
             "value": "2020.0528.1"
         },
-        "conParserImageTag": {
+        "nlpServicesImageTag": {
```

Comment:
did you change the image name itself in different PR?

Reply:
yes